src/data/nat/modeq.lean

```diff
@@ -398,6 +398,22 @@ by_contradiction $ λ hc,
   have (a + b) % c = a % c + b % c, from add_mod_of_add_mod_lt (lt_of_not_ge hc),
   by simp [dvd_iff_mod_eq_zero, *] at *
 
+lemma mod_mul_div_eq_div_mod (n a b : ℕ) : n % (a * b) / b = n / b % a :=
```

Comment:
I think for convenience it's maybe worth having `n % (b * a) / b = n / b % a` too, even though it follows trivially.